server.js

```diff
@@ -43,97 +40,90 @@ app.get('/', function homepage(req, res) {
-      {method: "DELETE", path:"/api/profile/:profile_id/projects/:_id", description:"deletes that project"},//done
-      {method: "PUT", path:"/api/profile/:profile_id/projects/:_id", description:"Updates that project by ID"},//done
-
-      {method: "GET", path: "/api/profile/:profile_id/vacation",description:"destinations i've visited"},//done
-      {method: "POST",path:"/api/profile/:profile_id/vacation",description:"Creates an entry for vacations under planning"}
-
+      { 
+        method: "GET", 
+        path: "/api", 
+        description: "Describes all available endpoints"
+      },
+      { 
+        method: "GET", 
+        path: "/api/profile", 
+        description: "Data about me"
+      },
+
+      { 
+        method: "POST",
+        path: "/api/profile/:profile_id/projects",
+        description:"creates new project"
+      },
+      { 
+        method: "DELETE", 
+        path:"/api/profile/:profile_id/projects/:_id", 
+        description:"deletes that project"
+      },
+      { 
+        method: "PUT", 
+        path:"/api/profile/:profile_id/projects/:_id", 
+        description:"Updates that project by ID"
+      },
+
+      { 
+        method: "GET", 
+        path: "/api/profile/:profile_id/vacation",
+        description:"destinations i've visited"
+      },
+      { 
+        method: "POST",
+        path:"/api/profile/:profile_id/vacation",
+        description:"Creates an entry for vacations under planning"
+      }
     ]
   })
 });
 
-/*End Point 2 - Get Profile Info*/
 app.get('/api/profile', function show(req, res) {
-  console.log("inside get profile",req.body);
   db.Profile.find({},function(err,new_profile){
     if(err) { console.log(" new profile not exist", err)}
     res.json(new_profile);
   })
 });
 
-/* End point 3 - Get All Projects */
-app.get('/api/profile/projects', function(req,res){
-  console.log("inside get projects", req);
-  db.Profile.find({},function(err,projects){
-    if(err){ console.log("Get All projects Method Error",err);}
-    res.json(projects);
-  })
-});
-
-/* End Point 4 - Create new project*/
 app.post('/api/profile/:profile_id/projects',function(req,res){
-  console.log(`inside create projects method ${req}`);
-  console.log(JSON.stringify(req.body));
-  db.Profile.findById(req.params.profile_id,function(err,foundProfile){
-
-  var newProject=new db.Projects(req.body);
+  db.Profile.findById(req.params.profile_id, function(err,foundProfile){
+    var newProject = new db.Projects(req.body);
 
-  foundProfile.projects.push(newProject);
-  foundProfile.save(function(err,savedprofile){
-    res.json(savedprofile);
+    foundProfile.projects.push(newProject);
+    foundProfile.save(function(err,savedProfile){
+      res.json(savedProfile);
+    });
   })
-})
 });
 
-/* End Point 5 - Show one project by ID*/
-app.get('/api/profile/:profile_id/projects/:_id',function(req,res){
-  console.log("inside getbyID");
-  db.Profile.findById(req.params._id, function(err, foundproj) {
```

Comment:
This does NOT show one project by ID. It gets all profiles. Your callback argument should be `profiles`, not `foundproj`. You also never need this endpoint, because all projects are already embedded in a profile. You also don't have a page where you just look at one project at a time.